### ISSUE TITLE
subdomain mapping?

### ISSUE THREAD

**Issue by IbrahimOuhamou** [jetzig-framework/jetzig#103]
how to implement subdomain mapping? like in 'username.github.io'? is that possible in normal views? or should I manually check for `request.headers.get("host")`?

**Comment by bobf** [comment]
Yes, currently you would need to use request headers, but I think it would be a nice feature to be able to map routes based on request host - I'll have a think about how to implement this and get back to you. It might be a while as I am busy with database integration stuff at the moment but we can use this issue to track it.

**Comment by IbrahimOuhamou** [comment]
@bobf 
thanks a lot. and for `/bismi_allah_route/{123}` we can just do it in the `bismi_allah.zig:get()` right?
like for `/user/{id}` we can use it in `user.zig:get()`
but what about `/user/{id}/{posts}`? is it in `user.zig:get() too`?
